cmd/telegraf/telegraf.go

```diff
@@ -315,6 +315,18 @@ func (t *Telegraf) runAgent(ctx context.Context, c *config.Config, reloadConfig
 		log.Printf("W! Deprecated secretstores: %d and %d options", count[0], count[1])
 	}
 
+	// Compute the amount of locked memory needed for the secrets
+	log.Printf("I! Managed secrets: %d", c.NumberSecrets)
+	required := 2 * c.NumberSecrets * uint64(os.Getpagesize())
+	available := getLockedMemoryLimit()
+	if required > available {
```

Comment:
Unlikely, but should this be `>=` to avoid a warning like "...4kb when 4kb is required."?